backend/app/transcendence/.eslintrc.js

```diff
@@ -0,0 +1,327 @@
+module.exports = {
+    "env":
+    {
+        "browser": true,
+        "es2021": true
+    },
+    "extends":
+    [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/recommended"
+    ],
+    "overrides":
+    [
+        {
+            "env": {
+                "node": true
+            },
+            "files": [".eslintrc.{js,cjs}"],
+            "parserOptions": {
+                "sourceType": "script"
+            }
+        }
+    ],
+    "parser": "@typescript-eslint/parser",
+    "parserOptions":
+    {
+        "ecmaVersion": "latest",
+        "sourceType": "module"
+    },
+    "plugins":
+    [
+        "@typescript-eslint"
+    ],
+    "rules":
+    {
+        // "indent":
+        // [
+        //     "error",
+        //     "tab"
+        // ],
+        // "linebreak-style": [
+        //     "error",
+        //     "unix"
+        // ],
+        "quotes":
+        [
+            "error",
+            "double"
+        ],
+        "semi":
+        [
+            "error",
+            "always"
+        ],
+        // custom rules based on 42
+        // POSSIBLE PROBLEMS
```

Comment:
Can you explicit possible problems?
Is it glitch issues, security issues? Is this something we should do differently and are aware of but this is in order to comply with the subject?

Reply:
below this line, these rules are used to prevent logic error into the code.
each rules documentation for example: https://eslint.org/docs/latest/rules/no-await-in-loop